src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -13147,6 +13147,172 @@ def test_aks_create_with_enable_advanced_network_observability(
             checks=[self.is_empty()],
         )
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17,
+        name_prefix="clitest",
+        location="eastus",
+    )
+    def test_aks_update_advanced_networking_observability_tls_management(
```

Comment:
maybe you could combine these test cases

Reply:
how would i be able to combine the tests if i am testing the create for 2 separate tls-management values?